app/src/androidTest/java/org/simple/clinic/sync/QuestionnaireResponseSyncIntegrationTest.kt

```diff
@@ -91,32 +75,18 @@ class QuestionnaireResponseSyncIntegrationTest {
         name = ""
     )
 
-    questionnaireSync = QuestionnaireSync(
-        syncCoordinator = SyncCoordinator(),
-        api = questionnaireSyncApi,
-        repository = questionnaireRepository,
-        lastPullToken = questionnaireLastPullToken,
-        config = config
-    )
-
-    questionnaireResponseSync = QuestionnaireResponseSync(
+    sync = QuestionnaireResponseSync(
         syncCoordinator = SyncCoordinator(),
-        api = questionnaireResponseSyncApi,
-        repository = questionnaireResponseRepository,
+        api = syncApi,
+        repository = repository,
         lastPullToken = lastPullToken,
         config = config
     )
   }
 
   private fun resetLocalData() {
-    clearQuestionnaireData()
     clearQuestionnaireResponseData()
     lastPullToken.delete()
-    questionnaireLastPullToken.delete()
-  }
-
-  private fun clearQuestionnaireData() {
```

Comment:
I see you've cleaned up functions which weren't of any use as of this moment. That's good. We'll add them later if needed.